src/io/darwin.zig

```diff
@@ -815,7 +822,9 @@ pub const IO = struct {
             .NOLCK => unreachable, // F_SETLK or F_SETLKW
             .OVERFLOW => return error.FileTooBig,
             .SRCH => unreachable, // F_SETOWN
-            .OPNOTSUPP => return error.OperationNotSupported, // not reported but need same error union
+
+            // not reported but need same error union
```

Comment:
At some later point, would be great to normalize our comments To start with a capital letter and end with a period.

Reply:
Indeed, that's always been the intention in TigerStyle, but perhaps could be made clearer?